src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -2597,6 +2595,10 @@ protected override void OnReceive(object message)
             else if (message is InternalClusterAction.InitJoinNack) { } //that seed was uninitialized
             else if (message is ReceiveTimeout)
             {
+                if (_attempts >= 2)
```

Comment:
Added some logging for nodes that were unable to perform a join to help troubleshoot the issue